dist/cleanup.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+// Retrieve environment variables and parameters
+const terraformCliPath = process.env.TERRAFORM_CLI_PATH;
```

Comment:
is this env guaranteed to be defined? Otherwise you better set default value to avoid most frequent JS runtime error of all times )
```js
const terraformCliPath = process.env.TERRAFORM_CLI_PATH || 'some default path`;
```